app/Http/Controllers/PasswordResetController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\ResetRequest;
+use App\Http\Requests\UpdateRequest;
+use App\Models\ResetPassword;
+use App\Services\UserService;
+use Carbon\Carbon;
+use App\Models\User;
+use Illuminate\Support\Facades\Mail;
+use App\Mail\ResetMail;
+
+class PasswordResetController extends Controller
+{
+    protected $userService;
+
+    public function __construct(UserService $userService)
+    {
+        $this->userService = $userService;
+    }
+
+    public function resetPassword(ResetRequest $request)
+    {
+        $email = $request->only('email');
+        $checkEmail = User::where('email', $email)->first();
```

Comment:
$checkEmail - contains instance of User class , confusing name. You should name it $user